src/core/logical_layers/renderers/stylesConfigs/mcda/mcdaStyle.ts

```diff
@@ -49,9 +49,8 @@ export function filterSetup(layers: MCDAConfig['layers']) {
 }
 
 export function linearNormalization(layers: MCDAConfig['layers']) {
-  const layersCount = layers.length;
-  if (layersCount === 1) {
-    return calculateLayer(layers.at(0)!);
```

Comment:
Old code was not dividing the result by coefficient (weight), which is a calculation error. 

MCDA formula is:
`V_total = Σ(V_norm(1) * weight(1), V_norm(2) * weight(2), V_norm(n) * weight(n)) / Σweight`